src/tabwidget.cpp

```diff
@@ -58,6 +59,7 @@ TabWidget::TabWidget(QWidget* parent) : QTabWidget(parent), tabNumerator(0), mTa
     connect(this, SIGNAL(tabCloseRequested(int)), this, SLOT(removeTab(int)));
     connect(tabBar(), SIGNAL(tabMoved(int,int)), this, SLOT(updateTabIndices()));
     connect(this, SIGNAL(tabRenameRequested(int)), this, SLOT(renameSession(int)));
+    connect(this, SIGNAL(tabTitleColorChangeRequested(int)), this, SLOT(setTitleColor(int)));
```

Comment:
As you're writing new codes, could you use the new Qt5 style of ```connect()```?

Reply:
Sure, done. Branch updated.

Reply:
S